calibration/CTDMOG/CGINS-CTDMOG-13603__20170401.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13603,CC_ptcb1,-1.246883e-004,
 37-13603,CC_ptcb2,0.00E+00,
 37-13603,CC_ptempa0,1.571107e+002,
-37-13603,CC_ptempa1,-7.169343e-002,
-37-13603,CC_ptempa2,8.220131e-007,
-37-13603,CC_wbotc,1.8600e-007,
+37-13603,CC_ptempa1,-7.169344e-02,
+37-13603,CC_ptempa2,8.220130e-07,
+37-13603,CC_wbotc,1.859974e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT